coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -57,6 +57,9 @@ def parse(self):
         elif self.language == "python" and self.docstyle == "doxygen":
             return self._parse_documentation_with_symbols(
                 ("@param ", " "), "@return ")
+        elif self.language == "java" and self.docstyle == "default":
+            return self._parse_documentation_with_symbols(
+                ("@param  ", " "), "@return ")
```

Comment:
that's the next goal, let us first hardcode that, translating this into a coalang system as soon as it gets implemented is easy^^